demo-project/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "semi": true,
```

Comment:
Can we set to 'false' even though it converts it?

Reply:
[9111a16](https://github.com/KILTprotocol/opendid/pull/55/commits/9111a169e73701e8e776465bd0b264cea312aba5)